packages/splits-sdk/src/client/liquidSplit.ts

```diff
@@ -177,60 +176,13 @@ class LiquidSplitTransactions extends BaseTransactions {
     return result
   }
 
-  // Graphql read actions
-  async getLiquidSplitMetadata({
-    liquidSplitAddress,
-  }: {
-    liquidSplitAddress: string
-  }): Promise<LiquidSplit> {
-    validateAddress(liquidSplitAddress)
-    const chainId = this._chainId
-
-    const response = await this._makeGqlRequest<{
-      liquidSplit: GqlLiquidSplit
-    }>(LIQUID_SPLIT_QUERY, {
-      liquidSplitAddress: liquidSplitAddress.toLowerCase(),
-    })
-
-    if (!response.liquidSplit)
-      throw new AccountNotFoundError(
-        'liquid split',
-        liquidSplitAddress,
-        chainId,
-      )
-
-    return await this.formatLiquidSplit(response.liquidSplit)
-  }
-
-  async formatLiquidSplit(
-    gqlLiquidSplit: GqlLiquidSplit,
-  ): Promise<LiquidSplit> {
-    this._requirePublicClient()
-    if (!this._publicClient) throw new Error()
-
-    const liquidSplit = protectedFormatLiquidSplit(gqlLiquidSplit)
-    if (this._includeEnsNames) {
-      await addEnsNames(
-        this._ensPublicClient ?? this._publicClient,
-        liquidSplit.holders.map((holder) => {
-          return holder.recipient
-        }),
-      )
-    }
-
-    return liquidSplit
-  }
-
   private async _requireOwner(liquidSplitAddress: string) {
     this._requireWalletClient()
 
     const liquidSplitContract = this._getLiquidSplitContract(liquidSplitAddress)
     const owner = await liquidSplitContract.read.owner()
 
-    // TODO: how to get rid of this, needed for typescript check
-    if (!this._walletClient?.account) throw new Error()
-
-    const walletAddress = this._walletClient.account.address
+    const walletAddress = this._walletClient!.account.address
```

Comment:
💯 

can't believe i didn't know about `!`

Reply:
chatGPT to the rescue